variables.tf

```diff
@@ -12,7 +12,7 @@ variable "type" {
   type        = string
   default     = ""
   validation {
-    condition     = var.type == "single" || var.type == "cluster" || var.type == "autoscaling"
+    condition     = var.type == "single" || var.type == "cluster" || var.type == "autoscaling" || var.type == "vpc"
```

Comment:
@iolesyk There is no use of `var.type == "vpc"` as a validation condition, because we only have to validate the variable values with `single`, `cluster` and `autoscaling`.